public/server/index.js

```diff
@@ -0,0 +1,22 @@
+
+
+
+
+
+
+
+
+
+
+// app.get("*", function (req, res) {
+
+
+//     console.log("something in route star")
+//     console.log("req.session", req.sessionID)
+//     if (req.sessionID) {
+//         console.log(__dirname)
+//         res.sendFile("/Users/edith/Desktop/movie-tinder-app/public/index.html");
```

Comment:
For production, I think we will be wanting to serve from the "build" folder because it will be a much smaller packet.  Also have you coordinated with Olivia on this?  It looks like this is all commented out for now though so I guess it doesn't matter yet.

Reply:
I just tried a bunch of stuff and forgot to remove it so I deleted it now